pybossa/util.py

```diff
@@ -358,11 +358,12 @@ class Pagination(object):
 
     """Class to paginate domain objects."""
 
-    def __init__(self, page, per_page, total_count):
+    def __init__(self, page, per_page, total_count, curr_page_count=0):
         """Init method."""
         self.page = page
         self.per_page = per_page
         self.total_count = total_count
+        self.curr_page_count = curr_page_count
```

Comment:
As discussed, this could be a property:

```python
@property
def curr_page_count(self):
    if self.has_next():
        return self.per_page
    else:
        return self.total_count % self.per_page
```
This way it would work with all pagination objects without additional changes (e.g. manual length computation)